controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz

    ik_impedance_controller:
      type: ik_impedance_controller/IkImpedanceController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


ik_impedance_controller:
  ros__parameters:
    k_gains:
      - 600.0
      - 600.0
      - 600.0
      - 600.0
      - 250.0
      - 150.0
      - 50.0
    d_gains:
      - 30.0
      - 30.0
      - 30.0
      - 30.0
      - 10.0
      - 10.0
      - 5.0